course_content/cartopy_course/cartopy_intro.ipynb

```diff
@@ -314,18 +456,34 @@
   {
    "cell_type": "code",
    "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
    "metadata": {
     "collapsed": true
    },
+   "source": [
+    "**NOTE:**\n",
+    "\n",
+    "For a worked solution, see [this example notebook](solutions/cartopy_final_exercise.ipynb)\n"
```

Comment:
Do you intend to keep this exercise in a separate notebook?

We should ideally keep consistent between the different courses. The Iris course (that we have reworked) currently loads the final exercise with 
```%load filename```

Reply:
> Do you intend to keep this exercise in a separate notebook? 

No, I think we should rework into a style consistent with the rest.
I just didn't get around to it yet.

I'll add this to my TODO list...
we should really build a common list of known outstanding tasks.